### DISCUSSION TITLE
Question about having drained event even when there is active job

### DISCUSSION THREAD

**Discussion by piotrekfracek** [OptimalBits/bull#2695] [Q&A] [answered]
Hey,
I have a question about `drained` event being fired when there is still an active job being processed.
Is that correct behavior? Shouldn't it be fired when there is nothing more to be processed and no active jobs?
Documentation is not clear about this.

**Comment by manast** [comment, marked as answer]
The drained event is fired when the wait list is empty regardless if there are active jobs or not.
